flytekit/remote/remote.py

```diff
@@ -788,6 +792,14 @@ def register_task(
         :param version: version that will be used to register. If not specified will default to using the serialization settings default
         :return:
         """
+        if serialization_settings is None:
```

Comment:
can you add a comment here explaining when this situation is relevant?  and maybe also a comment in tracker.py explaining when the second element in the return tuple (the thing that used to be "") is relevant?

Reply:
done